src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$day-width: 100px;
+$day-height: 100px;
+$calendar-padding: 10px;
+$color-day-background: #eee;
+$color-day-border: black;
```

Comment:
Instead of using 'black' as the border color, consider using a hexadecimal value (e.g. #000000) to maintain consistency with other color variables.